src/mgr.sol

```diff
@@ -70,38 +69,62 @@ interface RedeemLike {
 // not only DROP as an ERC20 balance in this contract, but also what's currently
 // undergoing redemption from the Tinlake pool.
 
-contract TinlakeManager is LibNote {
+contract TinlakeManager {
     // --- Auth ---
-    mapping (address => uint256) public wards;
-    function rely(address usr) external note auth { require(live, "TinlakeMgr/not-live"); wards[usr] = 1; }
-    function deny(address usr) external note auth { require(live, "TinlakeMgr/not-live"); wards[usr] = 0; }
+     mapping (address => uint256) public wards;
+    function rely(address usr) external auth {
+        require(live, "TinlakeMgr/not-live");
+        wards[usr] = 1;
+        emit Rely(usr);
+    }
+    function deny(address usr) external auth {
+        require(live, "TinlakeMgr/not-live");
+        wards[usr] = 0;
+        emit Deny(usr);
+    }
     modifier auth {
         require(wards[msg.sender] == 1, "TinlakeMgr/not-authorized");
         _;
     }
 
-    modifier ownerOnly {
-        require(msg.sender == owner, "TinlakeMgr/owner-only");
+    modifier operatorOnly {
+        require(msg.sender == operator, "TinlakeMgr/operator-only");
         _;
     }
 
-    // The owner manages the cdp, but is not authorized to call kick or cage.
-    address public owner;
+    // Events
+    event Rely(address usr);
+    event Deny(address usr);
+    event Draw(uint wad);
```

Comment:
nit: if possible make all the `uint` to `uint256`.

Reply:
Same for `int256`. I think some of them that we had changed, have been overwritten in this PR.